### PR TITLE
fix macos whl build

### PR DESCRIPTION
**Pull request by MediosZ**
1. $USER variable

the USER variable is not available in vs code terminal, see this [link](https://stackoverflow.com/questions/44572631/user-is-empty-when-running-bash-script-from-php-in-mac)
so set USER explicitly for compatibility

2. fix broken link

### CODE REVIEW

**Comment by chenqy4933** [comment]
LGTM

File: scripts/whl/macos/macos_whl_env_prepare.sh [hunk 1]
```diff
@@ -46,21 +46,21 @@ SRC_DIR=$($READLINK -f "`dirname $0`/../../../")
...
+            echo "FOUND install /Users/${USER}/.pyenv/versions/${pak} strip it..."
         else
             env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install ${pak}
         fi
         echo "###### do command: /Users/${USER}/.pyenv/versions/${pak}/bin/python3 -m pip install numpy wheel requests tqdm tabulate"
         /Users/${USER}/.pyenv/versions/${pak}/bin/python3 -m pip install numpy wheel
-        echo "###### do command: /Users/${USER}/.pyenv/versions/${pak}/bin/python3 -m pip install -r ${SRC_DIR}/python_module/requires-test.txt"
-        /Users/${USER}/.pyenv/versions/${pak}/bin/python3 -m pip install -r ${SRC_DIR}/python_module/requires-test.txt
+        echo "###### do command: /Users/$USER/.pyenv/versions/${pak}/bin/python3 -m pip install -r ${SRC_DIR}/imperative/python/requires-test.txt"
+        /Users/$USER/.pyenv/versions/${pak}/bin/python3 -m pip install -r ${SRC_DIR}/imperative/python/requires-test.txt
```
Comment (chenqy4933) [inline scripts/whl/macos/macos_whl_env_prepare.sh:63]: The requires-test.txt is just used in pytest, please change the requires-test.txt to requires.txt

File: scripts/whl/macos/macos_whl_env_prepare.sh [hunk 2]
```diff
@@ -46,21 +46,21 @@ SRC_DIR=$($READLINK -f "`dirname $0`/../../../")
...
     do
         echo "###### do command: env PYTHON_CONFIGURE_OPTS=\"--enable-shared\" pyenv install ${pak}"
-        if [ -e /Users/$USER/.pyenv/versions/${pak} ];then
-            echo "FOUND install /Users/$USER/.pyenv/versions/${pak} strip it..."
+        if [ -e /Users/${USER}/.pyenv/versions/${pak} ];then
+            echo "FOUND install /Users/${USER}/.pyenv/versions/${pak} strip it..."
         else
             env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install ${pak}
         fi
         echo "###### do command: /Users/${USER}/.pyenv/versions/${pak}/bin/python3 -m pip install numpy wheel requests tqdm tabulate"
```
Comment (chenqy4933) [inline scripts/whl/macos/macos_whl_env_prepare.sh:60]: please delete the line 60,61, it include in requires.txt

**Comment by chenqy4933** [comment]
LGTM

**Comment by genialwang** [comment]
Others, LGTM

**Comment by XindaH** [comment]
please solve conflicts and apply all changes as a single commit

**Comment by MediosZ** [comment]
please consider to merge it @XindaH

**Pull request closed without merging**